bin/edgex-docker-launch.sh

```diff
@@ -7,16 +7,37 @@
 #
 
 # Start EdgeX Foundry services in right order, as described:
-# https://wiki.edgexfoundry.org/display/FA/Get+EdgeX+Foundry+-+Users
+# https://docs.edgexfoundry.org/Ch-GettingStartedUsers.html
 
-COMPOSE_FILE=../docker/docker-compose.yml
-COMPOSE_URL=https://raw.githubusercontent.com/edgexfoundry/developer-scripts/master/compose-files/docker-compose-delhi-0.7.0.yml
+# EDGEX_COMPOSE_FILE overrides the default compose file. 
+# EDGEX_CORE_DB identifies the DB for Core Services.
+# EDGEX_SERVICES lists the service to start
+#
+# E.g. Start everything but metadata and data, use Redis for Core Services and a local compose file
+# EDGEX_SERVICES="logging command export-client export-distro notifications" \
+# EDGEX_CORE_DB=redis EDGEX_COMPOSE_FILE=docker/local-docker-compose.yml \
+# bin/edgex-docker-launch.sh
+
+if [[ -z $EDGEX_COMPOSE_FILE ]]; then
+  COMPOSE_FILE=$GOPATH/src/github.com/edgexfoundry/edgex-go/docker/docker-compose.yml
+  COMPOSE_URL=https://raw.githubusercontent.com/edgexfoundry/developer-scripts/master/compose-files/docker-compose-california-0.6.0.yml
```

Comment:
COMPOSE_URL is out of date. Should point to Delhi. See previous version.

Reply:
Done